doc/source/user_guide/migration.rst

```diff
@@ -0,0 +1,215 @@
+.. _ref_migration_guide:
+
+Migration
+#########
+
+In this section two guides are provided to help you migrate from PyVista plotters to the Ansys Tools Visualization Interface plotters.
+The first one addresses code migration, and the second one addresses documentation migration.
+
+Code migration guide
+====================
+
+This guide intends to help users transition from PyVista plotters to the new Ansys Tools Visualization Interface plotters. Since cases are very different
+from each other, a few examples are provided to cover the most common scenarios.
```

Comment:
```suggestion
Code migration
==========

This topic explains how to transition from PyVista plotters to the new Ansys Tools Visualization Interface plotters. Since cases vary greatly, a few examples are provided to cover the most common scenarios.
```